Change filename used for child multiplex QC report

allyhawkins opened the pull request:
In testing the latest changes in `scpca-nf`, I ran into an error where the filename had an extension of `.Rmd` rather than `.rmd` so it couldn't find the child template notebook to add in. I tried to change the filename, rather than change it to use `.Rmd`, but github wasn't noticing that as a change and letting me commit that... So here I'm updating the name of the template report. 

Here's a screenshot of the error I received: 
<img width="1149" alt="Screen Shot 2022-06-02 at 4 09 01 PM" src="https://user-images.githubusercontent.com/54039191/171738680-00f42721-5db2-4678-a2ac-60219e3b9e9b.png">

jashapiro commented:
Because `.rmd` is the extension we are using elsewhere, I'd rather leave the code and move the file. As you saw, git won't recognize case-only changes (on a case-insensitive file system like the mac), but you can get around this by telling git to do the renaming itself:

```
git mv inst/rmd/multiplex_qc.Rmd inst/rmd/multiplex_qc.rmd
```

allyhawkins commented:
Thanks for the suggestion @jashapiro. I reverted the change back to use the filename `multiplex_qc.rmd` and changed the file using the command you recommended so the only change in this PR is the file itself changing names.

jashapiro reviewed (approved):
⚡
